Sources/JXKit/JXContext.swift

```diff
@@ -20,21 +19,24 @@ open class JXContext {
             return scriptLoader.didChange != nil
         }
 
-        /// Whether `require` module support is enabled. Defaults to `true`.
-        public var moduleRequireEnabled: Bool
-        
         /// Configure a global script loader to use as the default when no loader is provided to the `Configuration`.
         ///
         /// - Seealso: ``JXContext/Configuration/scriptLoader``
         public static var defaultScriptLoader: JXScriptLoader = DefaultScriptLoader()
 
         /// The script loader to use for loading JavaScript script files. If the loader vends a non-nil `didChange` listener collection, dynamic reloading will be enabled.
         public var scriptLoader: JXScriptLoader
+
+        /// Configure a global logging function for JX log messages. Defaults to using `print`.
+        public static var defaultLog: (String) -> Void = { print($0) }
+
+        /// The logging function to use for JX log messages.
+        public var log: (String) -> Void
```

Comment:
Added very simple configurable log function that defaults to 'print', and now any messages we log (and by default our SwiftUI error handling) use this.

Reply:
How about `log: (Any...) -> Void`, which would let us use this for a shim for `console.log` as well?